sdk/kiosk/README.md

```diff
@@ -182,3 +202,77 @@ const withdraw = async () => {
 ```
 
 </details>
+
+<details>
+
+<summary>Create a Transfer Policy for a Type</summary>
+
+You can only create a TransferPolicy for packages that you own the `publisher` Object.
+
+It's recommended (unless you have a more advanced use case) to create only one transfer policy per type, or make sure that all transfer policies have the same rules.
+
+If you have multiple transfer policies, someone could purchase an item of that Type from a kiosk by using the easiest (to resolve) rules.
+
+You could extend the following snippet to do a check before creating the transfer policy, by using the `queryTransferPolicy` function, similar to the `purchaseAndResolvePolicies` example.
```

Comment:
As a best practice, you should use a single Transfer policy per type (T). If you use more than one policy for a type, and the rules for each differ, anyone that meets the conditions for any of the attached policies can purchase an asset from a kiosk. You can't specify which policy applies to a specific asset for the type when there is more than one policy attached. When someone meets the conditions that are easiest to meet, they are allowed to purchase and transfer the asset. 

Before you create a transfer policy, you can use the `quesryTransferpolicy` function to check the transfer policy associated with a type. This is similar to the `purchaseAndResolvePolicies` example.

Where is this example?